spec/system/admin/order_spec.rb

```diff
@@ -1005,9 +1005,9 @@ def serching_for_customers
     end
   end
 
-  describe "Legal Invoices" do
+  describe "Legal Invoices - Invoices feature enabled" do
     before do
-      Flipper.enable(:invoices)
+      Flipper.enable(:invoices, user)
```

Comment:
The `invoices` is supposed to be configured globally. 
I think there's no need to check for individual users